var/spack/repos/builtin/packages/py-corner/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCorner(PythonPackage):
+    """Make some beautiful corner plots."""
+
+    homepage = "https://corner.readthedocs.io"
+    pypi = "corner/corner-2.2.2.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("2.2.2", sha256="4bc79f3b6778c270103f0926e64ef2606c48c3b6f92daf5382fc4babf5d608d1")
+
+    depends_on("python@3.9:", type=("build", "run"))
+    depends_on("py-wheel", type="build")
```

Comment:
`py-wheel` can be removed since it is an automatic dependency of `PythonPackage`. You only have to specify it explicitly if you need a certain version of it.

Reply:
Thank you, I remove this dependence.